db/migrate/20230606133552_add_training_email_preferences_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTrainingEmailPreferencesToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :training_emails, :boolean
```

Comment:
What was the steer about default values? Presume we don't want any to trigger the registration but what about existing users. Will we use the "what's new" page?

Reply:
Returning users without email preferences will be redirected to the `email_preferences` page and asked for their preferences